src/test/request.test.js

```diff
@@ -407,3 +406,277 @@ describe('Manager get Pending requests', () => {
       });
   });
 });
+describe('Reject request', () => {
+  // reject request tests
+  let xToken;
+  let xRequestId;
+  let xUserToken;
+  before('signin the manager', (done) => {
```

Comment:
what about we use names like token, requestId, userToken instead.
- Also to use the hooks already in the test

Reply:
I have avoided using hooks in the test since sometimes those users ends up changing privileges while I need them to be specific.